### PR TITLE
Turned mpdfunk into mpdproxy

### LINKED ISSUE #16: Add Logging to mpd bot code

### LINKED ISSUE #24: Allow instance usage for mpd logic

### PR DESCRIPTION
**Pull request by camcast3**
Wrapped mpd logic into a new class structure so that it is a new instance for each stream bot we have using it.

### CODE REVIEW

File: musicplayer/mpdproxy.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# pylint: disable=no-member
+"""mpdfunk provides basic connectivity and song information from a locally running MPD instance"""
+import logging
+from mpd import MPDClient, MPDError
+
+# TODO(camcast): Need to run this code with close to prod environment to fully valdiate
```
Comment (camcast3) [inline musicplayer/mpdproxy.py]: Need to remove since I have already gone through and validate this new class structure.

File: musicplayer/mpdproxy.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# pylint: disable=no-member
+"""mpdfunk provides basic connectivity and song information from a locally running MPD instance"""
```
Comment (camcast3) [inline musicplayer/mpdproxy.py]: Should reword this to reflect the changes made to this file

**Review by prosolis — APPROVED** [review]
Thanks for adding error handling code. That is definitely something I should have done initially. :+1:

File: musicplayer/mpdproxy.py (hunk 2 above)
Comment (camcast3) [reply musicplayer/mpdproxy.py]: Done

File: musicplayer/mpdproxy.py (hunk 1 above)
Comment (camcast3) [reply musicplayer/mpdproxy.py]: Done
